### ISSUE TITLE
Installing Docker breaks Windows 10's Settings app

### ISSUE THREAD

**Issue by Doggie52** [docker/for-win#4018]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [ ] ~~I have uploaded Diagnostics~~ Not relevant?

### Expected behavior
Installing (and subsequently uninstalling) Docker leaves the computer in a usable state.

### Actual behavior
Installing Docker breaks the Windows 10 Settings app. In particular, the _Apps_ menu breaks. When clicked on, the window simply goes away. Event viewer produces the following log:

```
Faulting application name: SystemSettings.exe, version: 10.0.18362.1, time stamp: 0x066bf1a5
Faulting module name: KERNELBASE.dll, version: 10.0.18362.86, time stamp: 0xda9755c4
Exception code: 0xc000008c
Fault offset: 0x000000000003a839
Faulting process ID: 0x346c
Faulting application start time: 0x01d5195acbf0f095
Faulting application path: C:\Windows\ImmersiveControlPanel\SystemSettings.exe
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report ID: 5d39bd3c-45e7-42e0-9f57-849f0946007b
Faulting package full name: windows.immersivecontrolpanel_10.0.2.1000_neutral_neutral_cw5n1h2txyewy
Faulting package-relative application ID: microsoft.windows.immersivecontrolpanel
```

Uninstall does not help. Also, [_complete_ uninstall](https://success.docker.com/article/how-to-completely-remove-docker-in-windows-10) doesn't help. This included disabling Microsoft-Hyper-V and Containers.

I've got the .wer dump, happy to PM to someone who might want to look into this, don't want to share widely in case sensitive information is contained within.

### Information

  - Windows Version: 10.0.18362.116
  - Docker for Windows Version: don't know - latest downloaded 30 minutes ago from https://download.docker.com/win/stable/Docker%20for%20Windows%20Installer.exe

### Steps to reproduce the behavior

1. Install Docker for Windows.
2. Try to open the _Apps_ menu in _Settings_.
3. Uninstall Docker for Windows.
4. Try again.

**Comment by TheFreeman193** [comment]
Could your issue possibly be the same as this? [Click Start, Settings, Apps, crashes out settings in all builds 18xxx within 10 min after login](https://answers.microsoft.com/en-us/insider/forum/all/click-start-settings-apps-crashes-out-settings-in/0b892141-4eeb-41c1-927c-37b53f706a03?page=1)

I too noticed this issue on a vanilla 1903 build (18362.113), but the behaviour would vanish after 10-15 minutes. Could you test this immediately after startup, then again after 20 minutes or so? My guess would be some kind of dependency delay, and is supposedly fixed in build 18362.145.

**Comment by Doggie52** [comment]
Seen on another computer, unrelated to Docker. Thanks for pointing out, @TheFreeman193 !
